src/fu-util-common.c

```diff
@@ -1217,6 +1217,13 @@ fu_util_device_to_string (FwupdDevice *dev, guint idt)
 		fu_common_string_append_kv (str, idt + 1, _("Vendor"), tmp2);
 	}
 
+	/* branch */
+	if (fwupd_device_get_branch (dev) != NULL) {
+		/* TRANSLATORS: the stream of firmware, e.g. nonfree or open-source */
+		fu_common_string_append_kv (str, idt + 1, _("Branch"),
```

Comment:
As this is an attribute of the firmware installed not the device I wonder if you want to prefix this with "Firmware", IE `Firmware Branch`

Reply:
Or `Release Branch` perhaps?

Reply:
Yeah that's good

Reply:
Fixed.